packages/cli/src/lib/implementation/core-config.model.ts

```diff
@@ -20,6 +20,6 @@ export type ConfigCliOptions = {
   tsconfig?: string;
 };
 
-export type CoreConfigCliOptions = ConfigCliOptions &
-  PersistConfigCliOptions &
-  UploadConfigCliOptions;
+export type CoreConfigCliOptions = Omit<CoreConfig, 'upload'> & {
+  upload?: Partial<Omit<UploadConfig, 'timeout'>>;
+};
```

Comment:
We could add an `--upload.timeout` CLI argument, so that the config and CLI upload options are 1-to-1.

Reply:
Seems like a very specific argument that most people will not come across. 🤔 But you can create an issue for this and we can discuss it on the refinement.

Reply:
Now that I think about it, I'm not sure it helps much. Lets skip it until some user needs it.